.eslintrc.json

```diff
@@ -0,0 +1,117 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
+        "arrow-spacing": "error",
+        "block-scoped-var": "error",
+        "block-spacing": "error",
+        "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
+        "camelcase": "warn",
+        "comma-spacing": ["warn", { "before": false, "after": true }],
+        "comma-style": "error",
+        "consistent-return": "error",
+        "consistent-this": ["error", "that"],
+        "curly": ["error", "all"],
+        "default-param-last": "error",
+        "dot-location": ["error", "property"],
+        "dot-notation": "error",
+        "eol-last": "error",
+        "eqeqeq": ["error", "smart"],
+        "func-call-spacing": "error",
+        "generator-star-spacing": "error",
+        "grouped-accessor-pairs": ["error", "getBeforeSet"],
+        "implicit-arrow-linebreak": "error",
+        "indent": ["error", 4, { "SwitchCase": 1 }],
```

Comment:
don't trigger me. if this is an actual suggestion, 2 spaces, no tabs. it's enough to see what is going on without taking up too much space.

Reply:
you must get triggered a lot, cuz 4 spaces is by far the most popular one